### DISCUSSION TITLE
Switching versions shouldn't reset selector

### DISCUSSION THREAD

**Discussion by psarno** [primefaces/community#863] [PrimeNG] [answered]
Going from 17 to 16 should not take you back to the same page with 17 selected.

![image](https://github.com/primefaces/community/assets/18640860/8112a6b0-9ae7-4a5a-a184-470320ba39c7)

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Thanks a lot for the feedback, we'll address this issue in upcoming releases
